src/ray/raylet/worker_pool.cc

```diff
@@ -1148,8 +1150,8 @@ void WorkerPool::KillIdleWorker(std::shared_ptr<WorkerInterface> idle_worker,
           // kill the worker (e.g., when the worker owns the object). Without this,
           // if the first N workers own objects, it can't kill idle workers that are
           // >= N+1.
-          idle_of_all_languages_.push_back(
-              std::make_pair(idle_worker, last_time_used_ms));
+          idle_of_all_languages_.emplace_back(
```

Comment:
```suggestion
          idle_of_all_languages_.emplace_back(idle_worker, last_time_used)
```
to construct pair in place

Reply:
Sure, updated.